s3psetup/README.md

```diff
@@ -0,0 +1,28 @@
+### License
+
+Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+For any questions about this software or licensing,
+please email opensource@seagate.com or cortx-questions@seagate.com.
+
+## How to build and install s3psetup:
```

Comment:
what is s3psteup ? give one liner description. I would suggest to have script name as "s3setup"

Also I think to create the directory with name "s3setup" at scripts/provisioning/

And put all this python, readme files inside it.

Reply:
Reason using name "s3psetup" to distinguish from current shell script "s3_setup" ("p for python version"). Restructure the scripts might take some time due to rerun the builds and jenkins jobs. Will that be ok?

Reply:
yes, change the name now to "s3setup"